policy/README.md

```diff
@@ -0,0 +1,353 @@
+| `rules`         | Rule[]    | R   | List of applicable rule elements (see [“rule fields”](#rule-fields)) |
+
+Note that the provider-scope is at present not included.  A Provider should not be able to tell whether the policy is specific to them, or to a subset of Provider, or all Providers.
+
+<a name="rule-types"></a>
+### Rule Types 
+
+| Name    | Description |
+| ---     | --- |
+| `count` | Fleet counts based on regions.  Rule max/min refers to number of devices. |
+| `time`  | Individual limitations on time spent in one or more vehicle-states.  Rule max/min refers to number of minutes. |
+| `speed` | Global or local speed limits.  Rule max/min refers to miles per hour. |
+| `user`  | Information for users, e.g. about helmet laws.  Generally can't be enforced via events and telemetry. |
+
+<a name="rule-units"></a>
+### Rule Units 
+
+| Name      | Description |
+| ---       | --- |
+| `seconds` | Seconds |
+| `minutes` | Minutes |
+| `hours`   | Hours |
+| `mph`     | Miles per hour |
+| `kph`     | Kilometers per hour |
+
+<a name="rule-fields"></a>
+### Rule Fields 
+
+| Name         | Type     | R/O | Description      |
+| ---          | ---      | --- | ---              |
+| `name`       | String   | R   | Name of cap item |
+| `rule_type`  | enum     | R   | Type of policy (see [“rule types”](#rule-types))|
+| `rule_units` | enum     | O   | Measured units of policy (see [“rule units”](#rule-units))|
+| `geographies`| UUID[]   | R   | List of Geography UUIDs (non-overlapping) specifying the covered geography |
+| `statuses`   | Status[] | R   | Vehicle `status` to which this rule applies.  See [MDS Agency state diagram](https://github.com/CityOfLosAngeles/mobility-data-specification/blob/dev/agency/README.md#vehicle-events). |
+| `vehicle_types` | VehicleType[] | O | Applicable vehicle categories, default “all”.  See MDS shared data types document.  (link forthcoming) |
+| `minimum`    | integer | O | Minimum value, if applicable (default 0) |
+| `maximum`    | integer | O | Maximum value, if applicable (default unlimited) |
+| `start_time` | time    | O | Beginning time-of-day (hh:mm:ss) when the rule is in effect (default 00:00:00) |
+| `end_time`   | time    | O | Ending time-of-day (hh:mm:ss) when the rule is in effect (default 23:59:59) |
+| `days`       | day[]   | O | Days `["sun", "mon", "tue", "wed", "thu", "fri", "sat"]` when the rule is in effect (default all) |
+| `messages`   | { string:string } | O | Message to rider user, if desired, in various languages, keyed by language tag (see [Messages](#messages))|
+| `value_url ` | URL     | O | URL to an API endpoint that can provide dynamic information for the measured value (see [Value URL](#value-url)) |
+
+### Order of Operations
+
+Rules are ordered most-specific to most-general.  E.g. an “earlier” cap rule would take precedence over a “later” cap rule.  The internal mechanics of ordering are up to the Policy editing and hosting software.
```

Comment:
Seems like this should be enforced and standardized by the spec itself? unless there is some reason I'm not seeing here.

Reply:
We are using the fact that JSON has ordered lists to express the order in which the Rules should be evaluated.  Agree that the wording is a bit confusing, will fix that.